File: java/claude/P21.java
// Task p21: Read value from command line, add 1000, print result, return 0
public class P21 {
    public static void main(String[] args) {
        if (args.length > 0) {
            try {
                int value = Integer.parseInt(args[0]);
                int result = value + 1000;
                System.out.println(result);
            } catch (NumberFormatException e) {
                System.out.println("Invalid input.");
            }
        } else {
            System.out.println("Usage: java P21 <number>");
        }
    }
}

File: javascript/claude/p21.js
// Task p21: Read value from command line, add 1000, print result, return 0
if (process.argv.length > 2) {
  try {
    const value = parseInt(process.argv[2]);
    const result = value + 1000;
    console.log(result);
  } catch (e) {
    console.log("Invalid input.");
  }
} else {
  console.log("Usage: node p21.js <number>");
}
